.github/workflows/full-qa.yml

```diff
@@ -0,0 +1,135 @@
+name: Full QA
+
+on:
+  # new commits pushed to release branches
+  push:
+    branches:
+      - "master" # pushes to the `master` branch
+      - "*.*" # pushes to the released version branches (e.g. `1.0`)
+
+jobs:
+  unit-tests-p2:
+    name: Unit Tests (Polymer 2)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Node 12.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12.x
+
+      - name: Check out the source code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
This is only needed in the monorepo because there we need to detect packages changed in a PR since HEAD.

We can remove this for individual components where we always run all the tests.

Reply:
removed